### ISSUE TITLE
Status, notification and theme issues on iOS 16 with Mattermost 2.3.0

### ISSUE THREAD

**Issue by ent1ne** [mattermost/mattermost-mobile#7296]
#### Summary
There are a couple of issues with Mattermost app on Iphone with iOS 16 and Mattermost 2.3.0 installed

#### Steps to reproduce
1. Open app, set "online" status, minimize the app and then maximize it back
2. No specific steps to reproduce, it just happens sometimes
3. No specific steps to reproduce, it just happens sometimes

#### Expected behavior
1. Minimizing the app will keep the current status that will only change by timeout, but not immidiately
2. Notifications will always come for a reason and won't be empty or missing
4. Dark theme won't appear automatically if only it isn't set by user settings

#### Observed behavior (that appears unintentional) 
1. "Offline" status automatically sets after minimizing the app
2. Notifications and pushes come empty or sometimes don't come when they should (for example, when you've been mentioned in a thread or recieved a message)
3. Dark theme appears sometimes, but settings are set to always use only the light theme

**Comment by matthewbirtch** [comment]
> Themes are based on the user preference, at the moment there is no preference to set it to follow the system dark/light modes, I believe there was a feature for this being worked on but it was de-prioritized

Yes, there is a Jira ticket for this, but it's in the backlog with the team waiting to be prioritized.
https://mattermost.atlassian.net/browse/MM-39943

**Comment by ent1ne** [comment]
> The status won't change until the websocket disconnects which is 15 seconds after the app has been sent to the background or immediately if the app is fully closed.

So, there's no way to keep an online status when 15 seconds pass after sending the app to the background?

> can you expand on what does `empty` mean? also what are the user preferences around the notifications, as that can have an impact on when the notification is sent, perhaps you should take a look at the notifications.log file on the server to identify potential causes.

It's a lock screen app notification with no text in it. Being tapped it just goest to app after unlocking the screen — not any specific message or channel at all. Was wandering what could trigger such thing. 

Thanks!

**Comment by enahum** [comment]
> So, there's no way to keep an online status when 15 seconds pass after sending the app to the background?

as the websocket disconnects, the status changes.. but any other action in any other client will change the status back to online

> It's a lock screen app notification with no text in it. Being tapped it just goest to app after unlocking the screen — not any specific message or channel at all. Was wandering what could trigger such thing.

is it actually empty or it says something like "Notification" ? maybe is a notification setting on the phone to hide previews if the phone is not unlocked?
